CHANGELOG.md

```diff
@@ -18,7 +18,7 @@ UNRELEASED
 * Align language flags and translation status icons
 * Fix error when replacing media files without thumbnail
 * Add search function for media library
-
+* Fix error when deleting a poi that is used by an event
```

Comment:
Please add this to the `UNRELEASED` section and do just insert the line, not write the bullet point into an existing new line (the CHANGELOG is automatically parsed by our delivery tools, thus it needs a very fixed format of one new line after the release tag and two new lines after the bullet points...